### ISSUE TITLE
Improve dbfix to fix DDBBs with a type mismatch

### ISSUE THREAD

**Issue by kinow** [BSC-ES/autosubmit#1018]
In GitLab by @mcastril on Apr 20, 2023, 10:33

To be completed by @dbeltrankyl

**Comment by kinow** [comment]
In GitLab by @mcastril on Oct 24, 2023, 13:32

mentioned in issue autosubmit_api#34

**Comment by kinow** [comment]
@dbeltrankyl do you remember what this issue is about? The description is not very complete, I think. Thanks.

**Comment by dbeltrankyl** [comment]
No, but when 4.2 is merged, I think I'll open issues related to this, since 4.2 doesn't implement the dbfix right now. I think we can close this one in favour of the new one that will be open
